### PR TITLE
Add Fargate support to ECS types

### PR DESCRIPTION
**Pull request by ejholmes**
This updates the ECS Service type to support the new properties for enabling VPC networking and Fargate support.

### CODE REVIEW

**Comment by shyamvala** [comment]
Thanks for the update @ejholmes. To complete Fargate support I think some changes needed to be done on the Task Definition side as well especially for supporting CPU and Memory fields. 

http://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_TaskDefinition.html

**Comment by ejholmes** [comment]
Good call! I've updated, and add an example for launching nginx using Fargate (tested in CloudFormation).

File: troposphere/ecs.py [hunk 1]
```diff
@@ -197,9 +226,13 @@ class TaskDefinition(AWSObject):
 
     props = {
         'ContainerDefinitions': ([ContainerDefinition], True),
+        'Cpu': (basestring, False),
+        'ExecutionRoleArn': (basestring, False),
         'Family': (basestring, False),
+        'Memory': (basestring, False),
         'NetworkMode': (basestring, False),
         'PlacementConstraints': ([PlacementConstraint], False),
+        'RequiresCompatibilities': (list, False),
```
Comment (markpeek) [inline troposphere/ecs.py]: Instead of "list" this should be "[basestring]" which will do checks on the elements as well.

**Comment by ejholmes** [comment]
@markpeek fixed

**Comment by markpeek** [comment]
Thanks @ejholmes for the Fargate changes. And appreciate the input from @shyamvala as well.

**Pull request merged**